modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,241 @@
+    openshift.io/cluster-monitoring: "true"
+
+EOF
+----
+
+. Apply the operator group:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: ptp-operators
+  namespace: openshift-ptp
+spec:
+  targetNamespaces:
+    - openshift-ptp
+
+EOF
+----
+
+. Apply the subscription:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "${OC_VERSION}"
+  name: ptp-operator
+  source: "redhat-operators"
+  sourceNamespace: openshift-marketplace
+
+EOF
+----
+
+[id="cnf-applying-the-stream-control-transmission-protocol-patch_{context}"]
+== Applying the Stream Control Transmission Protocol (SCTP) patch
+
+Load and enable the SCTP kernel module on worker nodes in your cluster.
+
+.Procedure
+
+. Apply the SCTP machineconfig patch:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+  name: load-sctp-module
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/modprobe.d/sctp-blacklist.conf
+        - contents:
+            source: data:text/plain;charset=utf-8,sctp
+          filesystem: root
+          mode: 420
+          path: /etc/modules-load.d/sctp-load.conf
+
+EOF
+----
+
+[id="cnf-installing-the-sriov-network-operator_{context}"]
+== Installing the SR-IOV Network Operator
+
+Install the SR-IOV Network Operator by using the {product-title} CLI or the web console.
+
+. Apply the SR-IOV operator namespace:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-sriov-network-operator
+  labels:
+    openshift.io/run-level: "1"
+
+EOF
+----
+
+. Apply the SR-IOV operator group:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: sriov-network-operators
+  namespace: openshift-sriov-network-operator
+spec:
+  targetNamespaces:
+  - openshift-sriov-network-operator
+
+EOF
+----
+
+. Apply the SR-IOV operator subscription:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "${OC_VERSION}"
+  name: sriov-network-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+
+EOF
+----
+
+== Verifying your changes
+
+Use the following command to verify the changes have been applied to the cluster (the command will block until all the modifications are done):
```

Comment:
cluster. 

What does "the command will block" mean? Also, we avoid parentheticals, so when this sentence is revised I would make it a standalone sentence.

Reply:
Vitaly advises to omit the last sentence. Change made.